autoemulate/experimental/sensitivity_analysis.py

```diff
@@ -0,0 +1,264 @@
+from typing import Optional
+
+import pandas as pd
+from SALib.analyze.morris import analyze as morris_analyze
+from SALib.analyze.sobol import analyze as sobol_analyze
+from SALib.sample.morris import sample as morris_sample
+from SALib.sample.sobol import sample as sobol_sample
+
+from autoemulate.experimental.data.utils import ConversionMixin
+from autoemulate.experimental.emulators.base import Emulator
+from autoemulate.experimental.types import DistributionLike, NumpyLike, TensorLike
+
+# NOTE: we still use these functions from main
+# should we just move them to experimental as well?
+from autoemulate.sensitivity_analysis import (
```

Comment:
This is a good point as I don't think we have strategy yet for combining experimental and current un-refactored code - perhaps we should have an issue for it and approach this as a last step before 1.0 importing where we can for now?

Reply:
Yes having an issue for this would be useful.

Reply:
See #544